anchor-service/Cargo.toml

```diff
@@ -12,13 +12,19 @@ anyhow.workspace = true
 async-trait.workspace = true
 ceramic-core.workspace = true
 ceramic-event.workspace = true
+ceramic-sql.workspace = true
 cid.workspace = true
 expect-test.workspace = true
+futures.workspace = true
 indexmap.workspace = true
+ipld-core.workspace = true
 multihash-codetable.workspace = true
+multibase.workspace = true
 serde.workspace = true
-serde_ipld_dagjson.workspace = true
+sqlx.workspace = true
```

Comment:
do we need this dependency? Ideally we should export everything we need from the `ceramic-sql` crate.

Reply:
nah, this is fine. Since we moved all the queries into the service crates so it makes sense.